src/utils/image.utils.ts

```diff
@@ -149,7 +149,7 @@ const buildIpfsMediaUriByInfo = (
 
   if (useMediaHost && uri.startsWith('http')) {
     // This option also serves as a proxy for any `http` source
-    return `${MEDIA_HOST}/${size}/web/${uri.replace(/^https?:\/\//, '')}`;
+    return uri;
```

Comment:
The tests here https://github.com/madfish-solutions/templewallet-mobile/actions/runs/16049416801/job/45288273778 failed because of this change. I suppose we should add one parameter to this function to control using media host exactly here